src/pybind/mgr/dashboard_v2/frontend/src/app/ceph/performance-counter/table-performance-counter/table-performance-counter.component.html

```diff
@@ -0,0 +1,10 @@
+<cd-table [data]="counters"
+          [columns]="columns"
+          columnMode="flex"
+          [toolHeader]="false"
+          [footer]="false"
+          [limit]="0">
+  <ng-template #valueTpl let-row="row">
```

Comment:
Could you move this template to the data table templates?

Reply:
This is a very special cell template for the perf counter table, so i don't think we should move it to the 'generic' data table templates.